machine.go

```diff
@@ -31,7 +33,7 @@ type Machine struct {
 	MachineKey  string `gorm:"type:varchar(64);unique_index"`
 	NodeKey     string
 	DiscoKey    string
-	IPAddress   string
+	IPAddresses MachineAddresses
```

Comment:
This is a breaking change, needs to be documented in the changelog.

Reply:
Note of breaking change added to changelog.